### DISCUSSION TITLE
I'm wondering which fields the search covers

### DISCUSSION THREAD

**Discussion by ovizii** [mealie-recipes/mealie#585] [Q&A] [answered]
I tried searching for "egg" and found a couple of recipes in my mealie instance but some which contained "egg" in the Ingredients or Instructions were not found. 

Is this by design? 
edit: I'm running 0.5.1 via docker.

**Comment by hay-kot** [comment, marked as answer]
Only the title and description are currently used. The ingredient section is going through a refactor to add some major additional functionality. Once the core of that is nailed down I'll be looking at adding better search/filter based off ingredients. Ideally you'd be able to filter/search by ingredients the same as the categories. Then you could list out the ingredients you have and see what recipes you could make.
